write-fonts/src/tables/variations.rs

```diff
@@ -116,8 +116,15 @@ impl PackedDeltas {
             (MIN..=MAX).contains(&val)
         }
 
-        fn count_leading_zeros(slice: &[i16]) -> usize {
-            slice.iter().take_while(|v| **v == 0).count()
+        fn count_leading_zeros(slice: &[i16]) -> u8 {
+            let mut limit = u8::MAX as i32;
+            slice
+                .iter()
+                .take_while(|v| {
```

Comment:
This could just be
```take(u8::MAX as usize).take_while(…)```

Reply:
That's the missing limit() I couldn't find!!

Reply:
Fixed, ty